packages/quantic/force-app/examples/main/lwc/exampleRelevantGenerativeAnswering/exampleRelevantGenerativeAnswering.html

```diff
@@ -130,6 +130,7 @@
                   engine-id={engineId}
                   answer-style={answerStyle}
                   multiline-footer={multilineFooter}
+                  collapsible="true"
```

Comment:
```suggestion
                  collapsible
```

Reply:
Yea actually I think I'll remove this since the default should be false?

The property can still be found and used in the test version of the component, but the default search page shouldn't have a collapsible RGA. 

Thanks I forgot to remove it.